internal/github/find-pr_test.go

```diff
@@ -40,15 +40,13 @@ func TestFindPR_backport(t *testing.T) {
 	// https://github.com/elastic/beats/commit/fe25c73907336fc462d5e6e059d3cd86512484fe
 	res, err := github.FindPR(ctx, c, "elastic", "beats", "fe25c73907336fc462d5e6e059d3cd86512484fe")
 	require.NoError(t, err)
-	require.Len(t, res.Items, 4)
-	// not a backport: https://github.com/elastic/beats/pull/31396
-	require.Equal(t, 31396, res.Items[0].PullRequestID)
-	// backport: https://github.com/elastic/beats/pull/31417 => source: https://github.com/elastic/beats/issues/31013
-	require.Equal(t, 31013, res.Items[1].PullRequestID)
-	// backport: https://github.com/elastic/beats/pull/31396 => source: https://github.com/elastic/beats/issues/31369
-	require.Equal(t, 31369, res.Items[2].PullRequestID)
+	require.Len(t, res.Items, 3)
+	// backport: https://github.com/elastic/beats/pull/31482 => source: https://github.com/elastic/beats/pull/30979
+	require.Equal(t, 30979, res.Items[0].PullRequestID)
+	// backport: https://github.com/elastic/beats/pull/31572 => source: https://github.com/elastic/beats/pull/31531
+	require.Equal(t, 31531, res.Items[1].PullRequestID)
 	// backport: https://github.com/elastic/beats/pull/31343 => source: https://github.com/elastic/beats/pull/31279
-	require.Equal(t, 31279, res.Items[3].PullRequestID)
+	require.Equal(t, 31279, res.Items[2].PullRequestID)
```

Comment:
I checked the `testify` docs and it supports a [`ElementsMatch`](https://pkg.go.dev/github.com/stretchr/testify/require#ElementsMatch) so we don't have to worry about the items order.

```suggestion
	prIDs := []int{}
	for _, p := range res.Items {
		prIDs = append(prIDs, p.PullRequestID)
	}

	// backport: https://github.com/elastic/beats/pull/31482 => source: https://github.com/elastic/beats/pull/30979
	// backport: https://github.com/elastic/beats/pull/31572 => source: https://github.com/elastic/beats/pull/31531
	// backport: https://github.com/elastic/beats/pull/31343 => source: https://github.com/elastic/beats/pull/31279
	require.ElementsMatch(t, [30979, 31531, 31279], prIDs)
```

What do you think?